src/main/java/de/dennisguse/opentracks/util/PermissionRequester.java

```diff
@@ -57,7 +57,7 @@ private void requestPermission(ActivityResultCaller context, @Nullable Runnable
         locationPermissionRequest.launch(permissions.toArray(permissions.toArray(new String[0])));
     }
 
-    private static final List<String> GPS_PERMISSION = List.of(Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION);
+    private static final List<String> GPS_PERMISSION = List.of(Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_BACKGROUND_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION);
```

Comment:
Can we document that ACCESS_BACKGROUND_LOCATION is used by PublicApi.StartRecording?